pkg/action/build-iso.go

```diff
@@ -187,27 +186,30 @@ func (b *BuildISOAction) ISORun() error {
 }
 
 func (b *BuildISOAction) PrepareEFI(rootDir, uefiDir string) error {
-	return b.bootloader.InstallEFIFallbackBinaries(rootDir, uefiDir, b.spec.Label)
+	err := b.renderGrubTemplate(uefiDir)
```

Comment:
EFI partition was missing the grub.cfg, included in the ISO filesystem but not in the EFI (vfat) filesystem.